src/app/popup/popup.component.html

```diff
@@ -1,5 +1,7 @@
+<h1 mat-dialog-title>Export as HTML...</h1>
 <div mat-dialog-content>
-  <p>{{data.convertedHTML}}</p>
+    <pre class="prettyprint" *ngIf="data.convertedHTML !== '' ; else noValidMarkdown"><code class="language-html">{{data.convertedHTML}}</code></pre>
```

Comment:
Kathy says this specific library doesn't do exactly what's necessary, but it'll remain for now.